### PR TITLE
Feature/core 477/update documentations (1st round of update)

### PR DESCRIPTION
**Pull request by safydy-r**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,84 +1,129 @@
...
+|---------------------------------|----------------------------------------------------------------------------------------------------------------------|
+| builder.config.json             | a list of **all needed routes**, used by the `src\lib\mixins\Routes.ts` helper                                       |
+| src\lib\mixins\Routes.ts        | A route helper for all vue component to allow us calling something like <br/>`this.route('routeKey.something', ...)` |
+| vue.config.js                   | Mocked endpoints to simulate backend response (webpack-dev-server stub routes), eg: `/audiofiles/upload`             |
+| src\router\*                    | Vue router, to map routes & views                                                                                    |
+| docs\routes\openapi\routes.yaml | OpenAPI endpoints definition to explain complex endpoints                                                            |
+| docs\routes\markdown\routes.md  | Generated doc from the `routes.yaml` after calling `yarn md-docs`                                                    |
```
Comment (bulatgab) [inline README.md:44]: Good to have this summary 👍

File: README.md [hunk 2]
```diff
@@ -1,84 +1,129 @@
...
 - But when we integrate the flow-builder into other project, we may need to call a global Flow like `new global.Flow(...)`.
 
-## Usage
+### Storybook
+1. You only need to serve it with `yarn storybook:serve`
+2. Then follow the provided url, by default it should be `http://localhost:6006/` (but in case the port `6006` is non-available, it may use different one)
 
-### Create new flow
+### An explanation of app.config.json vs builder.config.json
+`app.config.json` is intended to support the configuration needed when integrating this flow building ui into a larger platform or application while `builder.config.json` is intended to configure those features which are core to the flow builder - the blocks enabled, languages or content types supported etc.
```
Comment (bulatgab) [inline README.md:99]: If you know about `app.config.json` and can add a bit more about it - maybe examples of what we are configuring - that would be nice. I still don't understand what we need it for 😄

File: docs/BUILDER_INTEGRATION.md [hunk 3]
```diff
@@ -365,8 +358,8 @@ import {
 ## Overriding default views.
 
 - If your project requires custom versions of what's in the `src/views/` directory, we encourage you to override the components you need to change in the manner described [here](#overriding-and-customising-non-block-components) rather than extending the existing views as syncing changes in the templates between flow-builder versions is onerous.
-    - For example, to customise the toolbar you can override the global toolbar component with a component that wraps this and uses it's slots to add custom buttons.
-- We hope to standardise slots in these views so that they can instead be wrapped by a custom version and select parts overriden in future.
+    - For example, to customise the toolbar you can override the global toolbar component with a component that wraps this and uses its slots to add custom buttons.
+- We hope to standardise slots in these views so that they can instead be wrapped by a custom version and select parts overridden in the future.
```
Comment (bulatgab) [inline docs/BUILDER_INTEGRATION.md:362]: Thank you for fixing typos 🙌

**Review by bulatgab — APPROVED** [review]

**Review by object-required — APPROVED** [review]

File: docs/BUILDER_INTEGRATION.md (hunk 3 above)
Comment (safydy-r) [reply docs/BUILDER_INTEGRATION.md:362]: you're welcome :D

**Pull request merged**